pyfr/integrators/dual/pseudo/densepseudoschemes/__init__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from pkg_resources import resource_listdir, resource_string
+import re
+
+
+def _get_coefficients(scheme):
+        a = []
+
+        scheme = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', scheme)
+        scheme = re.sub(r'\(|\)|,', '', scheme).strip()
+        scheme = scheme[1:-1] if scheme.startswith('[') else scheme
+
+        for l in scheme.splitlines():
+            if not l:
+                continue
+
+            a.append([float(f) for f in l.split()])
+
+        # The last row is the b vector
+        return a[:-1], a[-1]
+
+def get_densepseudoscheme(namein, porderin):
+    rpaths = resource_listdir(__name__, 'schemes')
+    scheme = None
+
+    for path in rpaths:
+        m = re.match(r'([a-zA-Z0-9\-~+]+)-s(\d+)'r'(?:-po(\d+))?\.txt$', path)
+        name, nstages, porder = m.group(1), m.group(2), m.group(3)
+        if name == namein and (int(porder) == porderin or porder == '*'):
```

Comment:
Don't use * for any porder; some systems such as Windows do not permit * in file names.  Just drop the -po suffix for a scheme which can work at any order.